packages/badge/src/FzBadge.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <div :class="classes">
+  <div :class="classes" class="h-20 flex items-center font-medium">
```

Comment:
there are already fixed classes in `classes`, I would keep them all in one place, either here or in `classes`